linker/jvm/src/main/scala/org/scalajs/linker/backend/closure/ClosureLinkerBackend.scala

```diff
@@ -138,8 +134,12 @@ final class ClosureLinkerBackend(config: LinkerBackendImpl.Config)
     }
 
     val content = new java.lang.StringBuilder
-    for (topLevelVarDecls <- topLevelVarDeclarations)
-      content.append(topLevelVarDecls + "\n")
+    for (topLevelVarDecls <- topLevelVarDeclarations) {
+      val varDecls =
+        if (!topLevelVarDecls.startsWith("let ")) topLevelVarDecls
+        else "var " + topLevelVarDecls.stripPrefix("let ")
+      content.append(varDecls + "\n")
+    }
```

Comment:
Oh, dear; what is happening here? Do you think we can improve this by passing on a js.Tree instead?

Reply:
Yeah, much better.